test/functional/tools/collection_paired_test.xml

```diff
@@ -1,6 +1,7 @@
 <tool id="collection_paired_test" name="collection_paired_test" version="0.1.0">
   <command>
     cat $f1.forward $f1['reverse'] >> $out1;
+    echo 'collection identifier: $f1.element_identifier'
```

Comment:
```suggestion
    echo 'Collection name: $f1.name'
```
It's equivalent but a better example, I think, since this is the collection name not a collection element identifier .